weblate/templates/configuration/custom.css

```diff
@@ -1,53 +1,104 @@
+{% load custom_filters %}
+
+{% with header_color_split=header_color|default:""|split:"," %}
+{% with header_text_color_split=header_text_color|default:""|split:"," %}
+{% with navi_color_split=navi_color|default:""|split:"," %}
+{% with nav_text_color_split=navi_text_color|default:""|split:"," %}
+{% with focus_color_split=focus_color|default:""|split:"," %}
+{% with hover_color_split=hover_color|default:""|split:"," %}
+
+body {
+  {% if header_color_split.0 %}
+    --header-color: {{ header_color_split.0 }};
+  {% endif %}
+  {% if header_text_color_split.0 %}
+    --header-text-color: {{ header_text_color_split.0 }};
+  {% endif %}
+  {% if navi_color_split.0 %}
+    --navi-color: {{ navi_color_split.0 }};
+  {% endif %}
+  {% if nav_text_color_split.0 %}
+    --navi-text-color: {{ nav_text_color_split.0 }};
+  {% endif %}
+  {% if focus_color_split.0 %}
+    --focus-color: {{ focus_color_split.0 }};
+  {% endif %}
+  {% if hover_color_split.0 %}
+    --hover-color: {{ hover_color_split.0 }};
+  {% endif %}
+}
+
+body[data-theme="dark"] {
+  {% if header_color_split.1 %}
+    --header-color: {{ header_color_split.1 }};
+  {% endif %}
+  {% if header_text_color_split.1 %}
+    --header-text-color: {{ header_text_color_split.1 }};
+  {% endif %}
+  {% if navi_color_split.1 %}
+    --header-text-color: {{ navi_color_split.1 }};
+  {% endif %}
+  {% if nav_text_color_split.1 %}
+    --header-text-color: {{ nav_text_color_split.1 }};
+  {% endif %}
+  {% if focus_color_split.1 %}
+    --header-text-color: {{ focus_color_split.1 }};
+  {% endif %}
+  {% if hover_color_split.1 %}
+    --header-text-color: {{ hover_color_split.1 }};
+  {% endif %}
+}
+
 {% if header_color %}
   .navbar-inverse {
-    background-color: {{ header_color }};
+    background-color: var(--header-color);
```

Comment:
What happens if variable is not set?

Reply:
Nothing, css ignores the rule. And the previous style will still be applied